crates/nu-utils/standard_library/tests.nu

```diff
@@ -2,7 +2,7 @@ def main [] {
     for test_file in (ls ($env.FILE_PWD | path join "test_*.nu") -f | get name) {
         let $module_name = ($test_file | path parse).stem
 
-        echo $"(ansi default)INFO  Run tests in ($module_name)(ansi reset)"
+        print $"(ansi default)INFO  Run tests in ($module_name)(ansi reset)"
```

Comment:
#8848 will do this.

Reply:
if i know my `git`, this should not bring any conflict between the two PRs :thinking:

Reply:
https://github.com/nushell/nushell/pull/8448/files#diff-5308521bd271aeed9d77fee7e0f64dfe125e10e2089bc965b8a6570504ea19ec

Reply:
oh you mean the `log info` and `log debug` :open_mouth: 

i think you are right!

Reply:
@presidento 
should be fixed in 78d3e2976fa5d427ad869664f10f9dd916635bfa :wink: